messages/2.8.0.md

```diff
@@ -0,0 +1,38 @@
+# MarkdownTOC - 2.8.0
+
+## Changes
+
+- Allow regular expressions in 'id_replacements' Ref: #113
+
+## Notice
+
+Setting format has been changed.
+You have to modify `id_replacements` section if you customised `MarkdownTOC.sublime-settings` by yourself.
```

Comment:
Suggestion: remove "by" , so it reads "`MarkdownTOC.sublime-settings` yourself."

Reply:
👍 thanks